website/content/en/docs/getting-started /_index.md

```diff
@@ -1,7 +1,10 @@
 ---
-title: Install the Operator SDK CLI
-linkTitle: Installation
-weight: 1
+title: "Getting Started"
```

Comment:
I agree with @jmrodri. 
The contact here is about how to install so IMO it should be `Installation`
The Getting Started content is in https://github.com/operator-framework/getting-started which I think we should add here. IMO we could just copy and paste now and then, do the follow-ups required to fix it as we wish or todo it in another PR. @dmesser @jmrodri   wdyt?